From 959b1e73a00a14ece4bf2b6b086d939fe4061f24 Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Fri, 28 Jan 2022 13:32:57 +0200
Subject: [PATCH] [SW-58071] gaudi3: update regmap register

Some of register names in specs are updated so align gaudi3
regmap file.

Change-Id: I2ebb7750d924bff19fa359828fec9ab43539f829
---

diff --git a/arch/gaudi3/arc/regmap.h b/arch/gaudi3/arc/regmap.h
index 1ad4b79..89e4974 100644
--- a/arch/gaudi3/arc/regmap.h
+++ b/arch/gaudi3/arc/regmap.h
@@ -57,7 +57,7 @@
 #define mmI2C_S_MSTR_IF_CORE_HBW_AXPROT_OVD	mmD0_I2C_S_MSTR_IF_AXPROT_HBW_ARPROT_0_OVRD
 
 #define mmCPU_HOT_RST			mmD0_PCIE_CORE_HOT_RST
-#define mmPCIE_RSP_OK			mmD0_PCIE_WRAP_PCIE_RSP_OK
+#define mmPCIE_RSP_OK			mmD0_PCIE_WRAP_PCIE_RSP_OK_0
 
 #define mmLINK_CAPABILITIES_REG			mmD0_PCIE_DBI_SIG_LINK_CAPABILITIES_REG
 #define mmLINK_CONTROL2_LINK_STATUS2_REG	mmD0_PCIE_DBI_SIG_LINK_CONTROL2_LINK_STATUS2_REG
